### PR TITLE
Add Test for A Simple Linode Block Storage PersistentVolumeClaim

### PR DESCRIPTION
**Pull request by faem**
### General:

* [ ] Have you removed all sensitive information, including but not limited to access keys and passwords?
* [ ] Have you checked to ensure there aren't other open or closed [Pull Requests](../../pulls) for the same bug/feature/question?

### Pull Request Guidelines:

1. [ ] Does your submission pass tests?
1. [ ] Have you added tests? 
1. [ ] Are you addressing a single feature in this PR? 
1. [ ] Are your commits atomic, addressing one change per commit?
1. [ ] Are you following the conventions of the language? 
1. [ ] Have you saved your large formatting changes for a different PR, so we can focus on your work?
1. [ ] Have you explained your rationale for why this feature is needed? 
1. [ ] Have you linked your PR to an [open issue](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)

### CODE REVIEW

File: e2e/test/framework/pvc.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+			Namespace: f.namespace,
+		},
+		Spec: core.PersistentVolumeClaimSpec{
+			AccessModes: []core.PersistentVolumeAccessMode{
+				core.ReadWriteOnce,
+			},
+			StorageClassName: &f.StorageClass,
+			Resources: core.ResourceRequirements{
+				Requests: core.ResourceList{
+					core.ResourceName(core.ResourceStorage): resource.MustParse("10Gi"),
```
Comment (aspsk) [inline e2e/test/framework/pvc.go]: Can you please add tests for three different sizes: 1Gi, 10Gi, 20Gi

**Comment by faem** [comment]
@aspsk Thank you. Please check now.

File: e2e/test/Makefile [hunk 2]
```diff
@@ -2,6 +2,7 @@ $(GOPATH)/bin/ginkgo:
 	GO111MODULE=off go get -u github.com/onsi/ginkgo/ginkgo
 
 ginkgo_test: $(GOPATH)/bin/ginkgo
+	GO111MODULE=on
```
Comment (aspsk) [inline e2e/test/Makefile]: This won't work as the statement is only effective on this particular line. You can just leave a comment that this target expects the `GO111MODULE=on`. If you decide to enable it by default, you can do something like this:
```
diff --git a/e2e/test/Makefile b/e2e/test/Makefile
index 7f4a00d..edb2574 100644
--- a/e2e/test/Makefile
+++ b/e2e/test/Makefile
@@ -1,3 +1,5 @@
+export GO111MODULE=on
+
 $(GOPATH)/bin/ginkgo:
        GO111MODULE=off go get -u github.com/onsi/ginkgo/ginkgo
 
```

Comment (faem) [reply e2e/test/Makefile]: okay

**Pull request merged**